components/common/Searcher.vue

```diff
@@ -0,0 +1,59 @@
+<template>
+  <v-col id="searcher">
+    <v-autocomplete
+      id="searcher__input"
+      :items="items"
+      background-color="white"
+      item-color="green"
+      placeholder="Введите фразу для поиска"
+      no-data-text="Результатов нет. Попробуйте переформулировать."
+      append-icon=""
+      dense
+      clearable
+      rounded
+    ></v-autocomplete>
+  </v-col>
+</template>
+
+<script>
+export default {
```

Comment:
We are using Typescript. To make sure everything is properly type-checked, use `lang="ts"` and `Vue.extend`
```suggestion
<script lang="ts">
import Vue from 'vue';

export default Vue.extend({
```